tests/query/query-ctor-005.phpt

```diff
@@ -3,19 +3,13 @@ MongoDB\Driver\Query construction with negative limit
 --FILE--
 <?php
 
+/* The server expects a non-negative limit, but neither PHPC nor libmongoc
+ * performs any validation. The option is passed as-is. */
```

Comment:
As mentioned in [this comment in PHPC-2440](https://jira.mongodb.org/browse/PHPC-2440?focusedCommentId=6733311&focusedId=6733311&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6733311), the server will raise an error. I decided to leave this test in place just to demonstrate that we no longer have validation.

I wasn't interested in adding a functional test for the server behavior.